packages/plugin-team/test/integration/spec/actions.js

```diff
@@ -0,0 +1,48 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '../..';
+
+describe(`plugin-team`, function() {
+  describe(`#addConversation()`, () => {
+    it(`adds an existing group conversation to a team`, () => {});
+  });
+
+  describe(`#addMember()`, () => {
+    it(`adds a team member to a team`, () => {});
+  });
+
+  describe(`#assignModerator()`, () => {
+    it(`assigns a team moderator`, () => {});
+  });
+
+  describe(`#archive()`, () => {
+    it(`archives a team`, () => {});
+    it(`archives a team conversation`, () => {});
+  });
+
+  describe(`#joinConversation()`, () => {
```

Comment:
This is the function used to add yourself to an open (team) conversation. This is a feature that only exists for team conversations and not of typical group conversations so in some ways it seems like it should be in the teams module... When implemented it'll be putting a POST request on the `teams` resource, too.  I see what you're saying, though... could fit in either place if you think about it.